docs-exp/auth-types.actioncodeurl.operation.md

```diff
@@ -4,6 +4,8 @@
 
 ## ActionCodeURL.operation property
 
+The action performed by the email action link. It returns from one of the types from [ActionCodeInfo](./auth-types.actioncodeinfo.md)

```

Comment:
does {@link ActionCodeInfo} work?

Reply:
looks like it works in the auth-types docs but not the auth docs, probably will self resolve once I add documentation for ActionCodeInfo in both